runtime/integration-tests/src/cases/lp/pool_management.rs

```diff
@@ -466,6 +466,7 @@ fn update_tranche_token_metadata<T: Runtime>() {
 		super::setup_currencies(evm);
 		super::setup_pools(evm);
 		super::setup_tranches(evm);
+		super::setup_market_ratios::<T>();
```

Comment:
How did it work before then?

Reply:
Before, it was not necessary to send `UpdateTrancheToken` before requesting a deposit. This was recently changed.